nasa-gibs/worldview
# WV-2403: Show All Event Tracks Feature

## Description

This feature adds the ability to view event tracks for all of the events on the map at the same time. This also adds a default layer (blue marble) when the events are activated in the sidebar. Clustering was removed on geographic projections for better visibility of tracks when several are grouped closely together. Event specific layers are now removed when deselecting an event as well. 

## How To Test

After checking out to the feature branch you will need to do a `npm run build` as the config files were. updated to include a default events layer.

The best way to test this feature would be to select events from both the map and sidebar while sometimes deselecting events from the sidebar and to make sure the following criteria is always met:

1. If events tab is active but no single event is selected, the default blue marble layer should appear.
2. No layer should appear above the blue marble layer unless it was specifically selected in the layers tab prior to activating the events tab and selecting an event.
3. Clicking any event from the map or sidebar should always show the event specific event layers and never the default blue marble layer.
4. Individual tracks should not disappear if the show all tracks option is selected and you unselect an individual event.
5. Leaving the events tab of the sidebar should make the blue marble layer invisible. 
6. Leaving the events tab with a single event selected should keep the event specific layers on.
